admin/class-smaily-for-wp-admin.php

```diff
@@ -63,10 +63,13 @@ public function enqueue_styles() {
 	 * Register the JavaScript for the admin area.
 	 *
 	 * @since 3.0.0
+	 * @since 3.1.4 Added a conditional check to load scripts only if user has 'edit_plugins' capability.
+	 * @since 3.1.5 Changed capability check to check administrator role instead of 'edit_plugins' capability.
```

Comment:
`@since` is used to denote the version number when the function (or callback) was first introduced or the API of the function (or callback) was changed. It shouldn't be used as an internal implementation changelog.

Reply:
Got it. https://github.com/sendsmaily/sendsmaily-wordpress-plugin/pull/145/commits/986a1c4d4a6a98b32056984fa1202b6c2e25cff2